code/game/objects/structures/lattice.dm

```diff
@@ -121,3 +121,181 @@
 		else
 			to_chat(user, "<span class='warning'>You need one floor tile to build atop [src].</span>")
 		return
+
+
+/obj/structure/lattice/lift
+	name = "lift lattice"
+	desc = "A lightweight support lattice. These hold lift platform together."
+	icon = 'icons/obj/smooth_structures/catwalk.dmi'
+	icon_state = "catwalk"
+	density = FALSE
+	anchored = TRUE
+	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 50)
+	max_integrity = 50
+	layer = LATTICE_LAYER //under pipes
+	plane = FLOOR_PLANE
+	canSmoothWith = list(/obj/structure/lattice/lift)
+	smooth = SMOOTH_MORE
+	//	flags = CONDUCT_1
+	obj_flags = CAN_BE_HIT | BLOCK_Z_OUT_DOWN
+	
+	var/list/lift_load = list() //things to move
+	var/datum/lift_master/LM
+
+
+/obj/structure/lattice/lift/Initialize(mapload)
+	. = ..()
+
+	RegisterSignal(src, COMSIG_MOVABLE_CROSSED, .proc/AddItemOnLift)
+	RegisterSignal(src, COMSIG_MOVABLE_UNCROSS, .proc/RemoveItemFromLift)
+
+/obj/structure/lattice/lift/proc/RemoveItemFromLift(datum/source, atom/movable/AM)
+	to_chat(world, "[src] RemoveItemFromLift([AM])")
```

Comment:
This message shown twice when AM moves out of platform. Don't know why.

Reply:
use debugger in vsc it should show you when it is called.

Reply:
Well since it shows twice it's probably called twice

Have you tried changing to UNCROSSED and CROSSED?

Reply:
Thanks, change to UNCROSSED can help.